index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
For reference, this is the layout before the change with the tagline under the logo: https://github.com/bazelbuild/bazel-website/pull/33

IMO having the tagline smaller than the "executive summary" makes it appear out of place, as if it is an afterthought, and I have not seen an example of this kind of pattern on other websites. Most other product websites I've seen have a concise tagline as the larger text and a brief summary in smaller text underneath it. Here are some examples:

- [Gradle](https://gradle.org/)
  * **Tagline**: "Accelerate developer productivity"
  * **Summary**: "From mobile apps to microservices, from small startups to big enterprises, Gradle helps teams build, automate and deliver better software, faster." 
- [Google AdSense](https://www.google.com/adsense)
  * **Tagline**: "Turn your passion into profit"
  * **Summary**: "AdSense is a free, simple way to make money online by placing ads on your website."
* [Google Cloud](https://cloud.google.com)
  * **Tagline 1**: "Build What's Next"
  * **Tagline 2**: "Better software. Faster."
  * **Summary** as 3 bullet points:
    * "Use Google's core infrastructure, data analytics and machine learning."
    * "Secure and fully featured for all enterprises."
    * "Committed to open source and industry leading price-performance."

In each of these cases, the main tagline is concise and memorable, and the brief summary that comes after them capture more details of the key value props.

IMO We should either:

1. Keep the tagline as the hero title, and perhaps make the size of the summary phrase larger so that it is not too much smaller than the tagline.
2. Replace our tagline with new one that is just as concise and memorable but would in some way describe Bazel better.
3. Omit the tagline completely.

As I said, I have not seen a case where the tagline of the product is smaller than the executive summary. If you are able to find some examples of well-designed sites of well-known projects that are able to do this, I would be happy to consider them. The examples I have included above include not only one of the most popular build systems in the Java world but also two of Google's most successful products, and all three of them follow the pattern that I am trying to implement here.